tapdb/sqlc/migrations/000010_multiverse_tree.up.sql

```diff
@@ -49,6 +49,12 @@ WHERE EXISTS (
     SELECT 1 FROM mssmt_roots WHERE namespace = 'multiverse-transfer'
 );
 
+-- Before we turn the existing universe_roots into multiverse_leaves, we need to
+-- fix a bug in the data where the asset ID was set for grouped assets.
+UPDATE universe_roots
```

Comment:
Will this resolve that issue where we saw we had two roots for an asset: with for the asset ID, and the other for the actual asset group?

Reply:
I thought so at first, but it turns out that was an issue with transfer proofs showing up in the stats. See fix here: https://github.com/lightninglabs/taproot-assets/pull/628.
So I'm going to remove that "asset ID fix" commit again, as it broke some queries as you mentioned below.
I'm just using `CASE WHEN ur.group_key IS NULL THEN ur.asset_id ELSE NULL END,` now for the `multiverse_leaves` insert.